console-subscriber/src/builder.rs

```diff
@@ -268,6 +274,14 @@ impl Builder {
         Self { self_trace, ..self }
     }
 
+    #[cfg(feature = "grpc-web")]
+    pub fn with_cors(self, cors: tower_http::cors::CorsLayer) -> Self {
+        Self {
+            cors_layer: Some(cors),
+            ..self
+        }
+    }
```

Comment:
I know that CORS is a common use case, but I think that we want to avoid exposing this directly in the public API.

One reason is that it adds types from (yet another) pre-1.0 crate to our public API, forcing us to release a breaking change every time that `tower-http` changes minor version. We already have this with `tonic` and it means that we have to bump our minor version every time that tonic does - we should avoid this if possible.

As we discussed on discord, I think that `into_parts()` is the correct solution to this, even if it does make things a bit longer.

On the other hand, we should definitely include a full set up including CORS in the example.

Reply:
Removed. Make sense!